packages/protocol/foundry.toml

```diff
@@ -11,6 +11,8 @@ optimizer_runs = 200
 ffi = true
 gas_limit = '18446744073709551615'
 memory_limit = 2073741824
+
+# David C: do we need to remove this as well?
```

Comment:
My opinion is let's remove this, which may make others think we have the same version of EVM on both L1 and L2.